plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/action/WorkloadManagementPlugin.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.wlm.action;
+
+import org.opensearch.action.ActionRequest;
+import org.opensearch.cluster.metadata.IndexNameExpressionResolver;
+import org.opensearch.cluster.node.DiscoveryNodes;
+import org.opensearch.common.inject.Module;
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.IndexScopedSettings;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.settings.SettingsFilter;
+import org.opensearch.core.action.ActionResponse;
+import org.opensearch.plugin.wlm.action.rest.RestDeleteQueryGroupAction;
+import org.opensearch.plugins.ActionPlugin;
+import org.opensearch.plugins.Plugin;
+import org.opensearch.rest.RestController;
+import org.opensearch.rest.RestHandler;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Supplier;
+
+/**
+ * Plugin class for WorkloadManagement
+ */
+public class WorkloadManagementPlugin extends Plugin implements ActionPlugin {
```

Comment:
I am taking that this class (and other duplicate files) will be removed from this since this is identical to one in PR: https://github.com/opensearch-project/OpenSearch/pull/14680/files#diff-f1059f230266d0d32d6e9ed96ba7be44008d74557715637b19926323199bd9f1

Reply:
We have this file in the other PR (we deleted the WorkloadManagementPluginModule file)

Reply:
We need this file for the UTs to run so we need it for now. But we'll remove duplicated files when merging